src/util-lua-sandbox.h

```diff
@@ -47,7 +47,7 @@ typedef struct SCLuaSbState {
     /* Execution Limits */
     uint64_t instruction_count;
     uint64_t instruction_limit;
-    uint64_t hook_instruction_count;
+    uint32_t hook_instruction_count;
```

Comment:
I see this used in calls to `lua_sethook` in which the arg is an `int`?
```
LUA_API void (lua_sethook) (lua_State *L, lua_Hook func, int mask, int count);
```

Reply:
so going for an int indeed and adding a comment